src/deploy/functions/backend.spec.ts

```diff
@@ -163,18 +131,14 @@ describe("Backend", () => {
           ],
           unreachable: ["region"],
         });
-        listAllFunctionsV2.onFirstCall().resolves({
-          functions: [],
-          unreachable: [],
-        });
```

Comment:
instead of deleting this, can we have it return empty run services (goes w/ changing runStub to reject in the before() setup